### PR TITLE
Update fhir.cerner.com for patient level private coverage

### PR DESCRIPTION
**Pull request by krvicky31**
Description
----
Need to update the Coverage Patch Implementation Note.

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.

Screenshot 2022-04-04 at 10 49 02 PM

- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/millennium/r4/financial/support/coverage.md [hunk 1]
```diff
@@ -265,7 +265,7 @@ _Implementation Notes_
 
 * This implementation follows the [JSON Patch](https://tools.ietf.org/html/rfc6902) spec.
 * Only operations on the paths listed below are supported.
-* For Private Coverages, only Encounter-level Coverages may be patched.
+* Patient-level private coverages are supported minus the relationship and subscriber replace operations.
```
Comment (SantanuBanerjee) [inline content/millennium/r4/financial/support/coverage.md:268]: Need confirmation from Alex , if we need to modify the language or add anything.

Comment (alexbeckner1) [reply content/millennium/r4/financial/support/coverage.md:268]: Let's update this to say "For Private Coverages, both Encounter-level and Patient-level Coverages may be patched, with the caveat of only supporting the /dependent and /period operations on Patient-level Coverages.

Comment (krvicky31) [reply content/millennium/r4/financial/support/coverage.md]: Updated the note: https://github.com/cerner/fhir.cerner.com/pull/820/files#diff-b606e00be5c937b992ec15dc4884b218073809d10c27279c771f072d522ded27R268

Comment (alexbeckner1) [reply content/millennium/r4/financial/support/coverage.md]: Thanks.

**Review by alexbeckner1 — APPROVED** [review]

**Review by BhavyaAcharya — APPROVED** [review]

**Review by SantanuBanerjee — APPROVED** [review]

**Review by aki-mathur — APPROVED** [review]

**Pull request merged**